FrontEnd/src/components/ProfileDetail/DetailedInfo/PhoneEmail.jsx

```diff
@@ -0,0 +1,104 @@
+import { useState } from 'react';
+import useSWR from 'swr';
+import { PropTypes } from 'prop-types';
+import classes from './PhoneEmail.module.css';
+
+function PhoneEmail ({ isAuthorized, profileId }) {
+    const [isPhoneShown, setPhoneShown] = useState(false);
+    const [isEmailShown, setEmailShown] = useState(false);
+    const authToken = localStorage.getItem('Token');
+
+    // TODO: change the logic of getting user.id when PR with hooks will be merged
+
+    const { data: userData } = useSWR(
+        authToken ? `${process.env.REACT_APP_BASE_API_URL}/api/auth/users/me/` : null,
```

Comment:
I think it is better to handle a 403 error instead of putting a null

Reply:
The useUser hook was created and this code was removed. (it was needed only to get user id)